docker/archlinux.dockerfile

```diff
@@ -45,20 +45,32 @@ pacman -Syu --disable-download-timeout --needed --noconfirm \
   git \
   libcap \
   libdrm \
-  namcap
+  namcap \
+  wget
 _DEPS
 
-# Setup builder user
 USER builder
 
+#Install Node
+# hadolint ignore=SC1091
+RUN <<_INSTALL_NODE
+#!/bin/bash
+set -e
+wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash
+source "$HOME/.nvm/nvm.sh"
+nvm install 20.9.0
+nvm use 20.9.0
+_INSTALL_NODE
+
 # copy repository
 WORKDIR /build/sunshine/
-COPY --link .. .
+COPY --link --chown=builder --chmod=777 .. .
 
 # setup build directory
 WORKDIR /build/sunshine/build
 
 # configure PKGBUILD file
+# hadolint ignore=SC1091
```

Comment:
This step is just configuring the PKGBUILD file. I don't believe node/npm would be needed here.

Sunshine build actually happens under `RUN <<_PKGBUILD`, which doesn't need most build deps pre-installed since the spec takes care of it.